heat/core/dndarray.py

```diff
@@ -928,6 +928,28 @@ def __eq__(self, other):
         """
         return relational.eq(self, other)
 
+    def isclose(self, other, rtol=1e-05, atol=1e-08, equal_nan=False):
+        """
+        Parameters:
+        -----------
+
```

Comment:
extra blank lines here between '------' and text

Reply:
Done